starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java

```diff
@@ -15,7 +15,9 @@
 import org.springframework.http.MediaType;
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
 
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
```

Comment:
I don't believe you need static in these imports

Reply:
Removed the "static" from the imports